### PR TITLE
FileSource

### PR DESCRIPTION
**Pull request by bstalk**
### FileSource
IQSample from the wave file.
Need to install libsndfile.
Please use -t filesource, and set -c filename=foo.wav
Currently supports only S24LE WAV format.

### CODE REVIEW

**Comment by jj1bdx** [comment]
@bstalk Looks like you still need to push commits in this pull request. I'd be glad if you let me know if everything looks OK on your side, by commenting on this issue. Thanks in advance! :)

**Comment by bstalk** [comment]
I finished to edit this feature PR. Would you please check them?

File: sfmbase/FileSource.cpp [hunk 1]
```diff
@@ -0,0 +1,376 @@
+// airspyhf-fmradion
+// Software decoder for FM broadcast radio with Airspy HF
+//
+// Copyright (C) 2015 Edouard Griffiths, F4EXB
+// Copyright (C) 2019 Kenji Rikitake, JJ1BDX
+//
```
Comment (jj1bdx) [inline sfmbase/FileSource.cpp]: @bstalk Add your Copyright notice here. You deserve to claim it.

File: include/FileSource.h [hunk 2]
```diff
@@ -0,0 +1,115 @@
+// NGSoftFM - Software decoder for FM broadcast radio with RTL-SDR
+//
+// Copyright (C) 2015 Edouard Griffiths, F4EXB
+//
```
Comment (jj1bdx) [inline include/FileSource.h]: @bstalk Add your copyright notice here. You deserve to claim it.

**Comment by jj1bdx** [comment]
@bstalk So far the code builds OK on macOS 10.14.6. Adding your copyright notice is the task left to merge this pull request.

**Comment by jj1bdx** [comment]
@bstalk Merged. Thanks!

**Pull request merged**